controllers/cindervolume_controller.go

```diff
@@ -774,22 +783,38 @@ func processCustomServiceConfig(customServiceConfig string) (bool, string) {
 		} else if strings.HasPrefix(token, "[") && strings.HasSuffix(token, "]") {
 			// Note the section name before looking for a volume_backend_name
 			sectionName = strings.Trim(token, "[]")
+
+			if isBackendSection && isLVM {
+				usesLVM = true
+				if targetIP != "" {
+					lvmTargetIP = targetIP
+				}
+			}
+			// Reset the target ip for the section
+			targetIP = ""
+			isBackendSection = false
+			isLVM = true
```

Comment:
This could use a comment. Initializing isLVM to True handles the fact that LVM is the default driver, so if no volume_driver is detected (at L803) then you must conclude it's an LVM backend. And if a volume_driver is detected at L803, the isLVM flag is properly updated at L805.

Reply:
Patch was removed comments no longer relevant.